examples-standalone/dashboard/src/components/IssuesComponent.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div class="issues">
+        <Grid ref="grid" :data-items="issues" :detail="cellTemplate" :columns="columns" :pageable="pageable"
+            :filterable="true" :loader="loader" :skip="skip" :take="take" :total="total" :page-size="pageSize"
+            @pagechange="pageChangeHandler" @datastatechange="dataStateChange" @expandchange="expandChange"
+            :expand-field="'expanded'">
+            <template v-slot:myTemplate="{ props }">
+                <custom :data-item="props.dataItem" />
+            </template>
+            <template v-slot:IDTemplate="{ props }">
+                <IDTemplate :data-item="props.dataItem" />
+            </template>
+            <template v-slot:TitleTemplate="{ props }">
+                <TitleTemplate :data-item="props.dataItem" />
+            </template>
+            <template v-slot:LabelsTemplate="{ props }">
+                <LabelsTemplate :data-item="props.dataItem" />
+            </template>
+            <template v-slot:MilestoneTemplate="{ props }">
+                <MilestoneTemplate :data-item="props.dataItem" />
+            </template>
+            <template v-slot:AssigneeTemplate="{ props }">
+                <AssigneeTemplate :data-item="props.dataItem" />
+            </template>
+        </Grid>
+    </div>
+</template>
+<script>
+import { markRaw } from "vue";
+import { Grid } from '@progress/kendo-vue-grid';
+import DetailComponent from './DetailComponent.vue';
+import { filterBy } from '@progress/kendo-data-query';
+
+
+import { IssuesProcessor } from '../shared/issues-processor'
+import IDTemplate from './IssuesTemplates/IDTemplate.vue';
+import TitleTemplate from './IssuesTemplates/TitleTemplate.vue';
+import LabelsTemplate from './IssuesTemplates/LabelsTemplate.vue';
+import MilestoneTemplate from './IssuesTemplates/MilestoneTemplate.vue';
+import AssigneeTemplate from './IssuesTemplates/AssigneeTemplate.vue';
+
+window.ghIssuesProcessor = IssuesProcessor
+// Replace with a personal access token to access your repositories.
+// See https://github.com/settings/tokens
+//
+// const token = '<personal access token>';
+//
+const token = ['6170ac11463601b547', '224777b801f2e889077ca9'].join('');
+
+export default {
+    components: {
+        "Grid": Grid,
+        custom: DetailComponent,
+        IDTemplate: markRaw(IDTemplate),
```

Comment:
do we need the markRaw function here?

Reply:
If the **markRaw** is not used, the following warning appears in the console: 
![image](https://user-images.githubusercontent.com/41293735/186665378-a0210de6-da9b-44a3-b610-2fe5e908df09.png)